test/unit/io/structure_file/structure_file_input_test.cpp

```diff
@@ -66,7 +66,10 @@ TEST_F(structure_file_input_class, construct_by_filename)
 {
     /* just the filename */
     {
-        test::tmp_filename filename = create_file("> ID\nACGU\n....\n");
+        test::tmp_filename filename{"structure_file_input_constructor.dbn"};
+        {
```

Comment:
please introduce a newline before beginning of scope

Reply:
:sleepy: Now I need to update the PR because of one newline. Is there anything else?